docs/index.rst

```diff
@@ -107,9 +107,10 @@ Contents:
 
    testing
    linters
-   testing-with-node
    testing-with-django
+   testing-with-node
    testing-with-casper
+   travis
```

Comment:
Not specific to this PR but it makes me notice that it's building up: the left nav is getting really long. Particularly unfortunate when using the left nav to browse sections that appear there below the fold -- when you get to the page, the part of the nav that you were just using disappears and you have to scroll down to get back to it.

I think probably the solution is another layer of collapsing detailed sections when not inside them -- e.g., "Code Testing" would just appear as one item in the left nav unless you're inside "Code Testing". Might motivate a bit of a round of tidying up the overall organization to ensure sections are still findable when their individual titles no longer appear on the nav.